dali/pipeline/operator/operator.cc

```diff
@@ -12,12 +12,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include "dali/pipeline/operator/builtin/split_merge.h"
 #include "dali/pipeline/operator/operator.h"
 
 namespace dali {
 
 template <typename Backend>
 void OperatorBase::EnforceUniformInputBatchSize(const Workspace &ws) const {
+  // Builtin operators have relaxed checks for the purpose of conditional execution
+  if (isSplitOrMerge(spec_.GetSchema())) {
```

Comment:
`IsSplit(schema) || IsMerge(schema)`?

Reply:
I would be happy to do something like `IsThisAnOpThatDoesn'tNeedThisCheck` but it's a bit long, so I used this one, as the Split and Merge go hand in hand.